cg/server/admin.py

```diff
@@ -454,6 +454,7 @@ class OrderView(BaseView):
 
     column_default_sort = ("order_date", True)
     column_editable_list = ["is_delivered"]
+    column_searchable_list = ["id", "ticket_id"]
```

Comment:
Is the search already done against the ticket id in the backend?

Reply:
This is the backend 💪